src/components/Calculator.js

```diff
@@ -27,25 +27,25 @@ function Calculator() {
   return (
     <div className="calculator">
       <div id="display" className="input-row">{display}</div>
-      <Item label="AC" cl="light-gray" objetChangeEvent={objetChangeEvent} />
-      <Item label="+/-" cl="light-gray" objetChangeEvent={objetChangeEvent} />
-      <Item label="%" cl="light-gray" objetChangeEvent={objetChangeEvent} />
-      <Item label="÷" cl="orange" objetChangeEvent={objetChangeEvent} />
-      <Item label="7" cl="light-gray" objetChangeEvent={objetChangeEvent} />
-      <Item label="8" cl="light-gray" objetChangeEvent={objetChangeEvent} />
-      <Item label="9" cl="light-gray" objetChangeEvent={objetChangeEvent} />
-      <Item label="x" cl="orange" objetChangeEvent={objetChangeEvent} />
-      <Item label="4" cl="light-gray" objetChangeEvent={objetChangeEvent} />
-      <Item label="5" cl="light-gray" objetChangeEvent={objetChangeEvent} />
-      <Item label="6" cl="light-gray" objetChangeEvent={objetChangeEvent} />
-      <Item label="-" cl="orange" objetChangeEvent={objetChangeEvent} />
-      <Item label="1" cl="light-gray" objetChangeEvent={objetChangeEvent} />
-      <Item label="2" cl="light-gray" objetChangeEvent={objetChangeEvent} />
-      <Item label="3" cl="light-gray" objetChangeEvent={objetChangeEvent} />
-      <Item label="+" cl="orange" objetChangeEvent={objetChangeEvent} />
-      <Item label="0" cl="double light-gray" objetChangeEvent={objetChangeEvent} />
-      <Item label="." cl="light-gray" objetChangeEvent={objetChangeEvent} />
-      <Item label="=" cl="orange" objetChangeEvent={objetChangeEvent} />
+      <ButtonItem label="AC" cl="light-gray" objetChangeEvent={objetChangeEvent} />
+      <ButtonItem label="+/-" cl="light-gray" objetChangeEvent={objetChangeEvent} />
```

Comment:
- [ ] This button is not working properly, kindly update it
![image](https://user-images.githubusercontent.com/62705947/190632122-47b942de-c389-471a-ab3f-91426b635d39.png)

Reply:
@GzimAsani , Thank you for the review feedback on the code. I just have one question: Can you please kindly more about the button issue you indicated on the screenshot above? which error did you get you pressed that button?

Reply:
@benardop give me a minute I will record it

Reply:
@GzimAsani Okay, thank you.

Reply:
https://www.loom.com/share/9ac0097ccac14f69adc20761b5f3b478
- I should be able to make the `8` to `-8` than again 8 and add or substract. Also I would suggest you add in fron of the number only `-` and remove it and not the whole `+/-` because it is kind of confusing.